src/AccessibilityInsights.Win32/Win32.csproj

```diff
@@ -14,12 +14,22 @@
     <BuildNumber>1.1.$([System.Math]::Round($([System.DateTime]::ParseExact($(CurrentDateString),`yyyy-MM-dd`,null).Subtract($(CountDate)).TotalDays)).ToString(`0000`))$(Revision)</BuildNumber>
   </PropertyGroup>
 
-  <Target Name="CreateVersionFile">
-    <WriteLinesToFile File="$(MSBuildProjectDirectory)\..\bld\A11yInsightsVersionInfo.cs" Overwrite="True" Lines="[assembly: System.Reflection.AssemblyVersion(&quot;$(BuildNumber)&quot;)] // Generated by build" />
+  <Target Name="CreateVersionFile" BeforeTargets="BeforeCompile">
+    <WriteLinesToFile File="$(MSBuildProjectDirectory)\..\bld\A11yInsightsVersionInfo.cs"
+                      Condition="!Exists('$(MSBuildProjectDirectory)\..\bld\A11yInsightsVersionInfo.cs')"
+                      Lines="[assembly: System.Reflection.AssemblyVersion(&quot;$(BuildNumber)&quot;)] // Generated by build"
+                      Overwrite="True" />
   </Target>
 
-  <Target Name="CreateAttributesInfoFile">
-    <WriteLinesToFile File="$(MSBuildProjectDirectory)\..\bld\A11yInsightsAttributesInfo.cs" Overwrite="True" Lines="[assembly: System.CLSCompliant(false)] // Generated by build" />
+  <Target Name="CreateAttributesFile" BeforeTargets="BeforeCompile">
+    <WriteLinesToFile File="$(MSBuildProjectDirectory)\..\bld\A11yInsightsAttributesInfo.cs"
+                      Condition="!Exists('$(MSBuildProjectDirectory)\..\bld\A11yInsightsAttributesInfo.cs')"
+                      Lines="[assembly: System.CLSCompliant(false)] // Generated by build"
```

Comment:
The target for `A11yInsightsAttributesInfo.cs` looks like it's just writing static content; can this one be replaced with a static file instead of dynamic build-time generation?

(or maybe just removed entirely? [Assemblies are non-CLS-compliant by default](https://learn.microsoft.com/en-us/dotnet/api/system.clscompliantattribute?view=net-6.0))

Reply:
Analyzer CA1014 wants this attribute to be explicitly set, so using the default value doesn't do the trick. I've moved this into a static file that gets automatically pulled into all shipping projects